backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ElasticSearchHealthCheck.java

```diff
@@ -76,9 +76,8 @@ public Result check() {
     Result toHealthCheckResult(ClusterHealthResponse response) {
         switch (response.getStatus()) {
             case GREEN:
-                return Result.healthy(COMPONENT_NAME);
             case YELLOW:
-                return Result.degraded(COMPONENT_NAME, response.getClusterName() + " status is YELLOW");
+                return Result.healthy(COMPONENT_NAME);
```

Comment:
So before people asks the question... Why YELLOW status is fine? According to this blog post http://chrissimpson.co.uk/elasticsearch-yellow-cluster-status-explained.html the yellow status means:

`Elasticsearch has allocated all of the primary shards, but some/all of the replicas have not been allocated.`

`Currently, our cluster health is yellow, meaning shard replicas have not been allocated. Why is this? This is because the current cluster only consists of a single node, so the replicas remain unasigned simply because no other node is available to contain them`

So yeah for testing we only have one node, so status will stay YELLOW, that's it...

Reply:
So if I understand well, yellow status is not fine, it's just fine for our test case, no? So why modifying production code?

Reply:
We consider we should support a single node ES in production as well.